libsolidity/ast/Types.cpp

```diff
@@ -2335,11 +2335,13 @@ bool StructType::isDynamicallyEncoded() const
 {
 	if (recursive())
 		return true;
-	solAssert(interfaceType(false).get(), "");
```

Comment:
ICE was coming from here, inLibrary is hardcoded to false here. Chenging to correct value for inLibrary made it pass here.

Reply:
Maybe if write solAssert(interfaceType(false).get(), interfaceType(false).message());, it throws a TypeError but doesn't make the ICE go away

Reply:
But if we comment down the line, the ICE shifts to solAssert(!containsNestedMapping(),""); Line 2630